ror/.stylelintrc.json

```diff
@@ -3,7 +3,11 @@
   "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
   "rules": {
     "at-rule-no-unknown": null,
```

Comment:
What about this entry?

Reply:
Thanks, @ahangarha, so acording to rails [docs](https://guides.rubyonrails.org/v5.1/asset_pipeline.html#:~:text=1%20What%20is,features%20in%20Rails.) every time
It compile all the `css` files into one big `scss` file where the directive will be.

I didn't find a reason to add this entry for now.

Reply:
I think we need to add thu, cuz later on when you use `@somthing` it will get errors.

Excuse my lack of experience with Tailwind, i have never used it before ^^!